source/LifecycleMethod.cpp

```diff
@@ -178,9 +244,9 @@ bool LifecycleMethod::validate(
     return false;
   }
 
-  for (const auto& callee : callees_) {
-    callee.validate(base_class, class_hierarchies);
-  }
+  // for (const auto& callee : callees_) {
+  //   callee.validate(base_class, class_hierarchies);
+  // }
```

Comment:
This still needs to be addressed. And same thing below.
You can do something like:
```
if (const auto* callees = std::get_if<std::vector<LifecycleMethodCall>>(callees_)) {
  // handle *callees
} else {
  const auto& graph = std::get< LifeCycleMethodGraph>(callees_);
  // handle graph
}
```